### PR TITLE
The great move

### PR DESCRIPTION
**Pull request by mpreisler**
I am unifying and fixing up all the templates, csvs, ...

Ideally after this we will only have a few well known locations.

`templates/static` is a little counter-intuitive and we may rename it. That is not the point of this PR. The point of this PR is to avoid having 4 different locations where OVAL files may appear.

### CODE REVIEW

**Comment by mpreisler** [comment]
TODO:
- move RHEVM3
- move SUSE
- move Ubuntu
- move Webmin
- move WRLinux
- remove support of the old OVAL locations from the build system

File: RHEL/7/templates/csv/oval_5.11/packages_removed.csv [hunk 1]
```diff
@@ -42,5 +43,5 @@ telnet-server
 tftp-server
 vsftpd
 xinetd
+yabrt
```
Comment (mpreisler) [inline RHEL/7/templates/csv/oval_5.11/packages_removed.csv]: Fixed

Comment (yuumasato) [inline RHEL/7/templates/csv/oval_5.11/packages_removed.csv]: A few lines above there is `pserv` which should be removed.

Comment (mpreisler) [inline RHEL/7/templates/csv/oval_5.11/packages_removed.csv]: Also fixed.

**Pull request merged**